ocfweb/docs/docs/staff/startertasks.md

```diff
@@ -43,48 +62,174 @@ you know how to use it:
 
 Once you’ve learned about `create`, you can start making changes to it!
 
-1. Follow the steps in the GitHub README to run the bot in development mode.
+1. Follow the steps in the [GitHub README](https://github.com/ocf/ircbot) to run
+   the bot in development mode.
 2. Make sure you can talk to the bot in development mode-- it will be named
    `create-yourusername` instead of simply `create`.
 3. Make a simple change and test that it works.
 4. Bonus: figure out how to get your development bot to join a public channel
    like #test. (there are multiple ways to do this!)
 
 ## Play with staff utilities\*
-OCF staff use a collection of scripts when interacting with the campus
-community. For example, before creating an account for a student organization,
-we make sure the person requesting the account is listed as a signatory for
-that group. Staff members use the `signat` command to perform this check.
+OCF staff use [[a collection of scripts|doc staff/scripts]] when interacting
```

Comment:
This link isn't particularly helpful, since it's a mostly empty page. A link to the `signat` docs would be more useful. We already have a link but I think it's fine to link it in the intro paragraph as well.

Reply:
I'll remove this link and link the signat mention later in the paragraph.